deploy/crds/base/jvmbuildservice.io_dependencybuilds.yaml

```diff
@@ -159,10 +159,14 @@ spec:
                     type: array
                   image:
                     type: string
+                  javaVersion:
```

Comment:
just making sure - you got this change by running `make generate` right @dwalluck 

thanks

Reply:
Yes. Same for the `config/kcp/*` changes.

Reply:
great thanks for confirming

marking resolved